lib/takeaway.rb

```diff
@@ -0,0 +1,30 @@
+require_relative './menu.rb'
+require_relative './order.rb'
+
+class TakeAway
+
+  attr_reader :order, :menu
+
+  def initialize
```

Comment:
This is another place where dependency injection can be used to decouple the classes.

`Takeaway` relies on the `Menu` and `Order` classes. We want to use dependency injection so that we can isolate the unit tests.
Without using dependency injection, our unit tests of the `Takeaway` class rely on the _actual_ implementation of the `Menu` and `Order` classes. This means if I break the `Menu` class, the `Takeaway` unit tests will also start failing. 

If instead, I've successfully isolated the unit tests by using doubles and stubs, if I break the `Menu` class, my unit tests for `Takeaway` should continue to pass, because in my unit tests, I am relying on doubles and stubs rather than the actual `Menu` class code .